ansible_runner/runner_config.py

```diff
@@ -84,7 +84,7 @@ def __init__(self,
                  resource_profiling_results_dir=None,
                  tags=None, skip_tags=None, fact_cache_type='jsonfile', fact_cache=None, ssh_key=None,
                  project_dir=None, directory_isolation_base_path=None, envvars=None, forks=None, cmdline=None, omit_event_data=False,
-                 only_failed_event_data=False):
+                 only_failed_event_data=False, containerized=False, container_runtime=False):
```

Comment:
Is it planned to add this to the CLI? I would like to get more documentation and schema around the options. See the ansible-builder PR https://github.com/ansible/ansible-builder/pull/16

That adds:

```
$ ansible-builder build --help
usage: ansible-builder build [-h] [-t TAG] [-f FILENAME] [-b BASE_IMAGE]
                             [-c BUILD_CONTEXT]
                             [--container-runtime CONTAINER_RUNTIME]

optional arguments:
  -h, --help            show this help message and exit
  -t TAG, --tag TAG     The name for the container being built.
  -f FILENAME, --file FILENAME
                        The definiton of the execution environment.
  -b BASE_IMAGE, --base-image BASE_IMAGE
                        The parent image for the execution environment.
  -c BUILD_CONTEXT, --context BUILD_CONTEXT
                        The directory to use for the build context. Defaults
                        to $PWD/context.
  --container-runtime CONTAINER_RUNTIME
                        Specifies which container runtime to use.
```

Where the use looks like:

```
$ ansible-builder build -c alan/bc/ -f alan/alan_ee.yml --container-runtime=docker
```

I would like to see some example commands for runner, and also start syncing the schema of these options between the two projects so that they are the same interface.

I could see the `--tag` option coming into play here.

Reply:
@AlanCoding to end game is for `ansible-builder` and `ansible-runner` to match the CLI interface for this, I have a PR that's waiting for this to merge that will "settle up" on that once I rebase.